persistence/src/main/scala/hmda/persistence/HmdaSupervisor.scala

```diff
@@ -0,0 +1,104 @@
+package hmda.persistence
+
+import akka.actor.{ ActorRef, ActorSystem, Props, Terminated }
+import hmda.api.processing.LocalHmdaEventProcessor
+import hmda.model.fi.SubmissionId
+import hmda.persistence.institutions.{ FilingPersistence, InstitutionPersistence, SubmissionPersistence }
+import hmda.persistence.processing.{ HmdaFileParser, HmdaFileValidator, HmdaRawFile, SingleLarValidation }
+
+object HmdaSupervisor {
+
+  case class FindActorByName(name: String)
+  case class FindFilings(name: String, institutionId: String)
+  case class FindSubmissions(name: String, institutionId: String, period: String)
+  case class FindProcessingActor(name: String, submissionId: SubmissionId)
+
+  def props(): Props = Props(new HmdaSupervisor)
+
+  def createSupervisor(system: ActorSystem): ActorRef = {
+    system.actorOf(HmdaSupervisor.props(), "supervisor")
+  }
+}
+
+class HmdaSupervisor extends HmdaActor {
+
+  import HmdaSupervisor._
+
+  var hmdaPersistentActors = Map.empty[String, ActorRef]
+
+  override def receive: Receive = {
+    case FindActorByName(name) =>
```

Comment:
I think I would find this a bit more readable if it were called something like `StartActorByName` instead of `Find`. And the same for `FindFilings`, `FindSubmissions`, and `FindProcessingActor`. If other people like this naming schema more readable I'm totally fine with it though :)